test/Microsoft.AspNet.Server.KestrelTests/KestrelServerInformationTests.cs

```diff
@@ -85,6 +85,38 @@ public void SetNoDelayUsingConfiguration()
             Assert.False(information.NoDelay);
         }
 
+
+        [Fact]
+        public void SetReuseStreamsUsingConfiguration()
```

Comment:
Please split into separate tests for `"true"` and `"false"`. Also please add tests for `null`, `string.Empty` and an invalid string (i.e. not parsable as `bool`).